selfdrive/car/body/carcontroller.py

```diff
@@ -14,8 +15,9 @@
 MAX_TURN_INTEGRATOR = 0.1  # meters
 
 
-class CarController:
+class CarController(CarControllerBase):
```

Comment:
Perhaps you make this a base class in a new PR, bring out common stuff like setting self.CP, etc.

Reply:
I started on this, but quickly came to the realization that it would duplicate >75% of this PR if I were to do it separately and type-hint as I went